Cargo.toml

```diff
@@ -11,4 +11,9 @@ license = "MIT OR Apache-2.0 OR Zlib"
 
 [dependencies]
 # wide = { path = "../wide" }
-wide = "0.4"
\ No newline at end of file
+wide = "0.4"
+serde = { version = "1.0", features = [], optional = true }
+serde_test = { version = "1.0", features = [], optional = true }
```

Comment:
It would be better to add `serde_test` as a `dev-dependency` instead and remove it here.